app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
The reason for not putting the switch in the partial itself is that I didn't want two levels of switching - it's clear coming to Detailed Guide that there's a big switch on the template. It'd be less clear if you came to a single Detailed Guide that the partials have switches buried in them, making it harder still to track down differences between views.